frontend/src/components/Documentation.vue

```diff
@@ -368,10 +364,10 @@
                 ). To clear the RNA levels, select the
                 <i>None</i>
                 option in the drop down list. RNA levels are available for both 2D and 3D Map
-                Viewer. The data is obtained from version 18 of the Protein Atlas with the units in
-                log
-                <sub>2</sub>
-                (TPM+1) associated with a gradient colorbar.
+                Viewer. The gene expression levels used are obtained from the file
+                <a href="https://www.proteinatlas.org/about/download" target="_blank">
+                  rna_tissue_hpa.tsv
+                </a>
```

Comment:
```suggestion
                </a>.
```

Reply:
Sounds a bit weird when one click the url for the file `rna_tissue_hpa.tsv` but guided to a download page in which the file can not be found without searching. 
What about 
```html
The gene expression levels used are obtained from the file <b>rna_tissue_hpa.tsv.zip</b> in the <a href="https://www.proteinatlas.org/about/download" target="_blank">download page</a> of the Protein Atlas.
```
?

Reply:
Nice suggestion! Implemented in 379c2781151774ae4e19ed36b172cc8813199cc7